modules/collect/src/main/java/com/opengamma/collect/named/ExtendedEnum.java

```diff
@@ -40,12 +37,24 @@
  * The configuration file also supports the notion of alternate names (aliases).
  * This allows many different names to be used to lookup the same instance.
  * <p>
- * The configuration file is found in the classpath.
- * It has the same location as the enum type and is a {@linkplain PropertiesFile properties file}.
- * Where multiple files are found in the classpath, a 'priority' value specifies which to load first
- * and a 'chain' flag specifies whether to chain to the next lower priority.
- * Provider classes are defined using the 'provider' key, where the value is the full class name of the provider.
- * Alternate names are defined using the 'alternate' key, where the value is 'AlternateName -&gt; StandardName'.
+ * The configuration file is found in the classpath. It has the same package location as the enum type
+ * and is a chained {@linkplain IniFile#ofChained(java.util.stream.Stream) INI file}.
+ * <p>
+ * A chained INI file allows multiple files to be on the classpath.
+ * A 'chain' section includes a 'priority' value to specify the order to load the files.
+ * The 'nextInChain' and 'removedSections' keys provide fine grained control.
+ * <p>
+ * Two sections control the loading of extended enum providers - 'providers' and 'alternates'.
+ * <p>
+ * The 'providers' section contains a number of properties, one for each provider.
+ * The key is the full class name of the provider.
+ * The value is either 'constants' or 'lookup'.
+ * A 'constants' provider defines the extended enums are public static constants.
```

Comment:
Would it be sufficient to check whether the class implements ``NamedLookup`` and if not assume the values are defined as constants?